src/test/resources/epub3/content-publication.feature

```diff
@@ -243,8 +246,7 @@ Feature: EPUB 3 ▸ Content Documents ▸ Full Publication Checks
 
   Scenario: Report an xpgt style sheet without a fallback
     See issues #271, #241
-    When checking EPUB 'content-xhtml-xpgt-no-fallback-error'
-    Then error CSS-010 is reported
+    When checking EPUB 'content-xhtml-link-no-fallback-valid'
     And no other errors or warnings are reported
```

Comment:
Same oddity of saying "And no other" when the error has been removed.

Line 130 at least says "And no errors", although I've gotten used to just seeing "Then no" when tests come back valid.

Reply:
👍 same as before, filed as #1386